storage/backup/backup-cli/src/lib.rs

```diff
@@ -17,8 +18,37 @@ use storage_client::{StorageRead, StorageReadServiceClient};
 
 pub type FileHandle = String;
 
+pub struct BackupService {
+    port: u16,
+    client: reqwest::Client,
+}
+
+impl BackupService {
+    pub fn new(port: u16) -> Self {
+        Self {
+            port,
+            client: reqwest::Client::new(),
+        }
+    }
+
+    async fn get_account_range_proof(&self, key: HashValue, version: Version) -> Result<Bytes> {
+        Ok(self
+            .client
+            .get(&format!(
+                "http://localhost:{}/state_range_proof/{:x}/{}",
+                self.port, key, version,
```

Comment:
Looking at the server implementation, should version be put before the key? Does it matter?